etc/schema.json

```diff
@@ -2360,37 +2370,82 @@
                 },
                 "http_content_type": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "The media type of the resource or data, the Content-Type header",
+                    "suricata": {
+                        "keywords": [
+                            "http.content_type"
+                        ]
+                    }
                 },
                 "http_method": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "The HTTP request method",
+                    "suricata": {
+                        "keywords": [
+                            "http.method"
+                        ]
+                    }
                 },
                 "http_port": {
-                    "type": "integer"
+                    "type": "integer",
+                    "description": "The port in the Host header if any"
                 },
                 "http_refer": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "An absolute or partial address of the web page that makes the request., the Referer header",
+                    "suricata": {
+                        "keywords": [
+                            "http.referer"
+                        ]
+                    }
                 },
                 "http_response_body": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "Base64 of the response body",
+                    "suricata": {
+                        "keywords": [
+                            "file.data"
+                        ]
+                    }
                 },
                 "http_response_body_printable": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "The ascii-printable characters of the response body",
+                    "suricata": {
+                        "keywords": [
+                            "file.data"
+                        ]
+                    }
                 },
                 "http_user_agent": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "a characteristic string that lets servers and network peers identify the application, operating system, vendor, and/or version of the requesting user agent, the User-Agent header",
+                    "suricata": {
+                        "keywords": [
+                            "http.user_agent"
+                        ]
+                    }
                 },
                 "length": {
-                    "type": "integer"
+                    "type": "integer",
+                    "description": "The response message length"
                 },
                 "org_src_ip": {
                     "type": "string"
```

Comment:
I have no idea what this is besides https://redmine.openinfosecfoundation.org/issues/2837

Reply:
Doesn't show up in any tests either.

Reply:
Side note : Commit 42adaf5627f09a41aefd19856ccea6d21f5d7df5 has no ticket ref, so I guess this comes from a private test